pyg_lib/ops/__init__.py

```diff
@@ -332,6 +331,62 @@ def index_sort(
     return torch.ops.pyg.index_sort(inputs, max_value)
 
 
+class Softmax(torch.autograd.Function):
+    @staticmethod
+    def forward(
```

Comment:
Can we define the autograd function directly in C++?

Reply:
Should be possible, will check.

Reply:
#282